src/stickerbook.js

```diff
@@ -581,6 +581,58 @@ class Stickerbook {
   }
 
   /**
+   * Places the current selected sticker programmatically on the canvas
+   * @param {Object} options An object with options for the placement
+   * @param {Number} options.x The x position at which to place the sticker
+   * @param {Number} options.y The y position at which to place the sticker
+   * @param {Number} options.xScale The x scale at which to place the sticker, defaults to 1
+   * @param {Number} options.yScale The y scale at which to place the sticker, defaults to 1
+   * @param {Number} options.rotation How much to rotate the image in degrees, defaults to 0
```

Comment:
is the convention that this is a clockwise rotation? Is that enough of a known convention for this kind of thing that we don't need to document it?